display/d.graph/d.graph.md

```diff
@@ -33,65 +33,60 @@ be used instead.
 
 The graphics language is simple, and uses the following commands:  
 
-\[ [\#](#comment) \| [move](#move) \| [draw](#draw) \|
-[polygon](#polygon) \| [polyline](#polyline) \| [color](#color) \|
-[text](#text) \| [size](#size) \| [symbol](#symbol) \|
-[rotation](#rotation) \| [icon](#icon) \| [width](#width) \]
-
-<span id="comment"></span>**\#** *comment*  
+**\#** *comment*  
 A line of comment which is ignored in the processing.
 
-<span id="move"></span>**move** *xpos ypos*  
+**move** *xpos ypos*  
```

Comment:
This means that we no longer support the anchors directly to the commands for existing URLs.

Just stating what it is doing here, if intended, simply skip.

Reply:
Right, that's intended.